data/json/items/magazine/50.json

```diff
@@ -44,5 +44,39 @@
     "reliability": 9,
     "reload_time": 110,
     "flags": [ "MAG_BULKY" ]
+  },
+  {
+    "id": "dsr50mag",
+    "type": "MAGAZINE",
+    "name": "DSR-50 Magazine",
+    "description": "3-Round box magazine for DSR-50.",
+    "weight": "470 g",
+    "volume": "500 ml",
+    "price": 15000,
```

Comment:
It shouldn't worth $150 after the Cataclysm. Add `price_postapoc`.

Reply:
how much should i put?

Reply:
Oh, I just noticed that all our magazines don't have postapoc price. Don't mind me.

Reply:
ahahhaha XD, that's why i confused about adding "price_postapoc" to the line (although it is recommended on a guide)

its okay, i already put it on the line.